### ISSUE TITLE
Rendering with multiple cameras

### ISSUE THREAD

**Issue by V-Klimov** [Unity-Technologies/PostProcessing#872]
##### What happened?
Modified example scene with from Unity with workbench.
The Main Camera has a PostProcessLayer component attached.
Do following
1. Create new Layer "Test"
2. Duplicate the workbench, move slightly and assing layer Test to it.
3. Remove layer Test from Main Camera culling mask.
4. Create a child Camera for Main Camera with the same position and direction. 
5. Set child camera culling mask to Test layer only, Clear Flags to 'Depth Only', Depth to '1'
Now things must run this way: Main Camera renders the whole scene except workbench clone with post-processing applied, then child camera renders workbench clone without post-processing.
But we see only workbench clone on the black background.
If we'll disable child camera -- whole scene (without workbench clone) with post effects appears.
If we'll disable PostProcessLayer on the Main Camera and enable child camera -- whole scene (with both workbenches) without post effects appears.
In other words -- child camera in some way makes post-processing produce a solid black screen.
How to fix it?

##### Post-processing stack version (v1, v2)?: 
V2

##### Unity version, operating system, target platform (standalone windows, mac, iOS, PS4...)?
2019.2.17f1, Windows10, Standalone Windows
